vars/pmmVersion.groovy

```diff
@@ -1,32 +1,51 @@
 def call(String type='latest') {
-  List<String> oldVersions = ['2.9.1', '2.10.0', '2.10.1', '2.11.0', '2.11.1', '2.12.0', '2.13.0', '2.14.0']
+  // List<String> oldVersions = ['2.9.1', '2.10.0', '2.10.1', '2.11.0', '2.11.1', '2.12.0', '2.13.0', '2.14.0']
+  List<String> oldVersions = ['2.12.0', '2.13.0']
+  List<String> oldVersions = []
```

Comment:
I don't get it. First you are setting oldVersion to '2.12.0' and '2.13.0' and then setting to an empty array?

Reply:
it was a typo - fixed now.